WcaOnRails/app/views/competitions/_competition_info.html.erb

```diff
@@ -75,6 +75,12 @@
       </dd>
     </dl>
 
+	<dl class="dl-horizontal">
```

Comment:
The indentation looks screwy here?

Reply:
Sorry, I was using a not-that-great editor.